### ISSUE TITLE
How to align two temporal dimensions?

### ISSUE THREAD

**Issue by 95AliceHong** [q1xiangchen/VMPs#2]
I found the content of the vmps module in your project and referenced it in slowfast.py. The content of forward in the SlowFastLoader class is:
def forward(self, x):
    assert(len(x.shape)==5)
    x_f=x[:]
    x_s=x[:,:,::self.alpha]
    return (x_s, x_f)
I added a line 'x_f, _=self. vmps (x_f)'
Then when I run it, the 'x=torch. cat (pool_out, dim=1)' in head_ helper. py gives an error saying that the time dimensions of two tensors are not consistent. One is the original number of  frames, and the other is missing one. I want to know how to align the two dimensions? Looking forward to your reply, thank you!

**Comment by q1xiangchen** [comment]
Thank you for your interest! Please refer to the [instructions](https://github.com/q1xiangchen/VMPs/blob/main/configs/TRAIN.md#finetuning-the-timesformer-with-vmps) section here, we actually require one more frame to get the VMPs. For further technical details, please check the paper.
